tools/plugins/com.liferay.ide.project.ui/META-INF/MANIFEST.MF

```diff
@@ -25,6 +25,9 @@ Require-Bundle: biz.aQute.bndlib;bundle-version="[4,5)",
  org.eclipse.jface.text,
  org.eclipse.jst.j2ee,
  org.eclipse.jst.server.core,
+ org.eclipse.lsp4e,
```

Comment:
Are these bundles available in the default Eclipse installation?  of not, we are going ot need to separate these out into separate feature even, that would have additional dependencies bundled in that feature/product, like we do elsewhere.

Reply:
These bundles are not in the default Eclipse installation, we will add them to feature.